src/plugins/liveobjects/liveobjectspool.ts

```diff
@@ -27,7 +27,7 @@ export class LiveObjectsPool {
    */
   deleteExtraObjectIds(objectIds: string[]): void {
     const poolObjectIds = [...this._pool.keys()];
-    const extraObjectIds = this._client.Utils.arrSubtract(poolObjectIds, objectIds);
+    const extraObjectIds = poolObjectIds.filter((x) => !objectIds.includes(x));
```

Comment:
this is good and will be slightly more performant. would be good actually to change the util too

Reply:
`Utils.arrSubtract` is not used anywhere it seems, would you prefer to update it to have this .filter/.includes and keep it, or just remove `Utils.arrSubtract` at this point?

Reply:
ah right, yeah i'd say just remove it 👍

Reply:
removed in https://github.com/ably/ably-js/pull/1898/commits/b73a15cf34b876ba4c6f1375c5f425cea18f6dbc